services/wfm/src/main/java/org/openkilda/wfm/topology/reroute/bolts/RerouteBolt.java

```diff
@@ -98,6 +99,17 @@ protected void handleInput(Tuple tuple) {
             emitRerouteCommands(tuple, inactiveFlows, message.getCorrelationId(),
                     rerouteInactiveFlows.getReason());
 
+        } else if (commandData instanceof RerouteAllFlowsForIsl) {
+            RerouteAllFlowsForIsl rerouteAllFlowsForIsl = (RerouteAllFlowsForIsl) commandData;
+            PathNode source = rerouteAllFlowsForIsl.getSource();
+            PathNode destination = rerouteAllFlowsForIsl.getDestination();
+            Collection<String> inactiveFlows =
+                    rerouteService.getAllFlowIdsForIsl(source.getSwitchId(), source.getPortNo(),
+                                                       destination.getSwitchId(), destination.getPortNo());
+
+            emitRerouteCommands(tuple, inactiveFlows, message.getCorrelationId(),
```

Comment:
The only difference I see between RerouteAffectedFlows and RerouteAllFlowsForIsl is that the latter causes reroute for inactive flows too. Am I wrong?

Reply:
No, you are right.